phd2.appdata.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2017 Mattia Verga <mattia.verga@tiscali.it> -->
+<component type="desktop-application">
+    <id>phd2.desktop</id>
+    <metadata_license>CC0-1.0</metadata_license>
+    <project_license>BSD-3-Clause</project_license>
```

Comment:
The LICENSE.txt file distributed in sources is a BSD-3-Clause, so I put that in <project_license>.

The <metadata_license> only refers to the appdata file itself.